### ISSUE TITLE
Charm list import

### ISSUE THREAD

**Issue by Fuuzetsu** [breakds/monster-avengers#5]
We can add charms/amulets but if we refresh, they're all gone. Ideally we'd be able to store and edit a file locally then import it in the web UI when doing searches.

**Comment by breakds** [comment]
Yep, I totally agree. I am thinking of using a google account of the users to store user-specific data actually, although that would be very difficult for Chinese users to do so ...

**Comment by Fuuzetsu** [comment]
I think depending on an external service is an overkill. A simple CSV file import/export should be more than enough.

I think some existing software (ASS?) already uses some simple file format. I don't use that software but if it's simple enough then it might be worthwhile adopting it so that users can easily migrate.

**Comment by breakds** [comment]
Yep. I would definitely like to do that. 
Have you tried Ping's Dex yet? It does support saving charms~

**Comment by Fuuzetsu** [comment]
> Have you tried Ping's Dex yet? It does support saving charms~

No, I don't use Windows and even if it runs in `wine`, I don't want to use proprietary software if I can at all avoid it. The license on this project is a big plus for me.

**Comment by breakds** [comment]
I have the same feeling towards windows as well. Ping is actually a friend of mine, and I think he has plan to port it to other platforms. Mono sounds promising since Dex is written in C#.
